shell/ci/release/confluence-publish.sh

```diff
@@ -75,4 +75,5 @@ while read -r file; do
   else
     info_sub "no space directive found, skipping ${file}"
   fi
-done < <(git diff --name-only HEAD HEAD~1 -- '*.md')
+  previousTag=$(git describe --abbrev=0 --tags "$(git rev-list --tags --skip=1 --max-count=1)" || git rev-list --max-parents=0 HEAD)
+done < <(git diff --name-only HEAD.."$previousTag" -- '*.md')
```

Comment:
i think this soln is vulnerable to the same thing this PR was aiming to prevent - https://github.com/getoutreach/devbase/pull/148

i get where this is going, but we'll need to modify it slightly to also fix what that PR I linked fixes too

Reply:
I learned way more than I ever wanted to know about git hashes, but I think I got a solution that covers all edge cases and isn't too messy.